lib/process_win32.cpp

```diff
@@ -3,6 +3,30 @@
 
 #include "rtl_exec.h"
 
+struct ThreadInfo {
+    HANDLE pHandle;
+    std::vector<unsigned char> *buffer;
+};
+
+static DWORD WINAPI reader_thread(LPVOID lpParam)
+{
+    ThreadInfo *ti = reinterpret_cast<ThreadInfo*>(lpParam);
+
+    for (;;) {
+        char buf[1024];
+        DWORD n;
+        if (ti->pHandle != INVALID_HANDLE_VALUE) {
```

Comment:
Don't need to check for invalid handle here.

Reply:
Fixed